docs/src/reference/gremlin-applications.asciidoc

```diff
@@ -1076,7 +1076,8 @@ Gremlin-Console |PLAIN SASL (username/password) |3.0.0-incubating
 |Pluggable SASL |3.0.0-incubating
 |GSSAPI SASL (Kerberos) |3.3.0
 |Gremlin.NET |PLAIN SASL |3.3.0
-|Gremlin-Python |PLAIN SASL |3.2.2
+1.2+v|Gremlin-Python |PLAIN SASL |3.2.2
```

Comment:
Did you mean to have that "1.2+v" in there that way? Won't that break the table structure?